From fef3c184a1e657c9ceb36b901310a2a0f1a28eaa Mon Sep 17 00:00:00 2001
From: Vlad Yasevich <vyasevic@redhat.com>
Date: Fri, 29 Aug 2014 19:01:21 -0400
Subject: [net] bridge: fix build

Message-id: <1409338885-30414-10-git-send-email-vyasevic@redhat.com>
Patchwork-id: 89902
O-Subject: [RHEL7 PATCH 09/13] net: bridge: fix build
Bugzilla: 896669
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Jiri Benc <jbenc@redhat.com>
RH-Acked-by: Hannes Frederic Sowa <hannes@redhat.com>

From: Alexei Starovoitov <ast@plumgrid.com>

fix build when BRIDGE_VLAN_FILTERING is not set

Fixes: 2796d0c648c94 ("bridge: Automatically manage port promiscuous mode")

Signed-off-by: Alexei Starovoitov <ast@plumgrid.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
(cherry picked from commit d4f0e0958dbacaa1c678d88bf02a4794c66fb832)
Signed-off-by: Vladislav Yasevich <vyasevic@redhat.com>
Signed-off-by: Jiri Benc <jbenc@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/net/bridge/br_private.h b/net/bridge/br_private.h
index 73ec0fc..e4bb7a4 100644
--- a/net/bridge/br_private.h
+++ b/net/bridge/br_private.h
@@ -759,7 +759,7 @@ static inline u16 br_get_pvid(const struct net_port_vlans *v)
  return VLAN_N_VID; /* Returns invalid vid */
 }
 
-static inline int br_vlan_enabled(struct net_bridge *br);
+static inline int br_vlan_enabled(struct net_bridge *br)
 {
  return 0;
 }
-- 
1.7.1